wp-parsely.php

```diff
@@ -339,13 +355,30 @@ public function insert_parsely_page() {
             $tags = array_unique($tags);
 
             $parselyPage['@type']          = 'NewsArticle';
+            $parselyPage['mainEntityOfPage'] = array(
+                '@type' => 'WebPage',
+                '@id' => get_permalink()
+            );
             $parselyPage['headline']       = $this->get_clean_parsely_page_value(get_the_title());
             $parselyPage['url']            = get_permalink();
             $parselyPage['thumbnailUrl']   = $image_url;
-            $parselyPage['articleId']      = $postId;
+            $parselyPage['image']          = array(
+                '@type' => 'ImageObject',
+                'url' => $image_url
+            );
             $parselyPage['dateCreated']    = gmdate('Y-m-d\TH:i:s\Z', get_post_time('U', true));
+            $parselyPage['datePublished']  = gmdate('Y-m-d\TH:i:s\Z', get_post_time('U', true));
+            $parselyPage['dateModified']   = gmdate('Y-m-d\TH:i:s\Z', get_the_modified_date('U', true));
             $parselyPage['articleSection'] = $category;
+             $parselyPage['author']        = array(
+                '@type' => 'Person',
+                'name' => $authors
```

Comment:
Did some Google sleuthing but couldn't come up with anything more definitive than [this StackExchange post](http://webmasters.stackexchange.com/questions/90187/schema-org-book-with-multiple-authors-using-json-ld), which recommends passing a list of author objects to a single `author` key, so I went that route. 

I guess we could test this by adding the updated code in this PR to a sample WP blog that uses our plugin and see if Google validates it okay before we merge in this repo.